backup/moodle2/restore_kanban_stepslib.php

```diff
@@ -139,6 +139,10 @@ protected function process_card($data): void {
         $data->originalid = $this->get_mappingid('kanban_card_id', $data->originalid);
         $data->createdby = $this->get_mappingid('user', $data->createdby);
 
+        if (empty($data->number)) {
+            $data->number = $DB->get_field('kanban_card', 'MAX(number)', ['kanban_board' => $data->kanban_board]) + 1;
```

Comment:
In boardmanager.php function get_next_card_number() wird der Fall, dass man ggf. keine Antwort bekommt abgedeckt. Hier nicht. Warum ist es hier nicht nötig?

Reply:
Dort wird die Möglichkeit mit abgedeckt, dass das Board nicht existiert, das der Funktion übergeben wird.